compliance/evidence.py

```diff
@@ -76,27 +76,47 @@ def from_evidence(cls, evidence):
             new_evidence.set_content(evidence.content)
         return new_evidence
 
+    @classmethod
+    def lazy_load(cls, path):
+        return LazyLoader(path, cls)
+
     @classmethod
     def from_locker(cls, path, locker):
         path = substitute_config(path)
         return cls(locker.get_evidence(path))
 
+    @property
+    def rootdir(self):
+        raise NotImplementedError('The root directory is not set.')
+
+    @property
+    def dir_path(self):
+        return str(PurePath(self.rootdir).joinpath(self.category))
```

Comment:
https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.as_posix instead of the cast to string?

Reply:
I thought that part of the reason for using pathlib.PurePath was to begin to address the lack of support on Windows systems.  So I used `str` here.  If that's not the case then I can change it to `.as_posix`.

Reply:
Ah yes good point.